SU2_CFD/src/solver_direct_heat.cpp

```diff
@@ -1238,11 +1264,10 @@ void CHeatSolverFVM::Heat_Fluxes(CGeometry *geometry, CSolver **solver_container
             thermal_conductivity = config->GetThermalDiffusivity_Solid()*rho_cp_solid;
           }
 
-          HeatFlux[iMarker][iVertex] = thermal_conductivity*dTdn*config->GetTemperature_Ref()*Area;
+          HeatFlux[iMarker][iVertex] = thermal_conductivity*dTdn; // TDE *config->GetHeat_Flux_Ref();
```

Comment:
I'll check. Do we agree that heat fluxes should be passed between zones in units of W/m² ? (It would make sense I think as different zones could eventually use different reference values?)

Reply:
Just another question concerning the reference heat flux. E.g. regarding this line from the `CIncNSSolver::Friction_Forces` routine, 
`HeatFlux[iMarker][iVertex] = -thermal_conductivity*GradTemperature*RefHeatFlux;` -
do I understand correctly that the idea behind `RefHeatFlux` is to undo all changes of units and magnitudes that might have been applied to `thermal_conductivity` and `GradTemperature` so that we don't have to care about them separately anymore?

Reply:
The idea on that line is to re-dimensionalize the value of the heat flux, since we always print it to the output in dimensional format. In the fluid solver, assuming we run in non-dim. mode, we have already applied non-dim. to all intermediate quantities in the code within the SetNondimensionalization() routine (here, thermal_conductivity and Temperature). We also compute the RefHeatFlux there, so that we can undo the non-dim. like we see in that routine.

I think we should do similarly anytime we want to output quantities like heat flux, and your proposal to always pass dimensional values across interfaces like CHT also sounds reasonable to me. We should also double-check what is done for FSI. It is important that we define a standard and follow it consistently.

Reply:
Okay, thanks for the clarification.

Then the changes in my latest commits do what they should :)